lib/plugins/aws/lib/naming.js

```diff
@@ -46,6 +46,10 @@ module.exports = {
 
   // Stack
   getStackName() {
+    if (this.provider.serverless.service.provider.stackName &&
+        this.provider.serverless.service.provider.stackName.length > 0) {
```

Comment:
Why do you check length? This should set string value, so I think would be good to use `_.isString(this.provider.serverless.service.provider.stackName)` instead of `this.provider.serverless.service.provider.stackName.length > 0`

Reply:
good call. updated code and pushed changes